platform/ios/test/MapViewTests.m

```diff
@@ -424,6 +424,29 @@ - (void)testInsetMapView {
                   @"compass should lie inside shrunken map view");
 }
 
+- (void)testContentInsetsWithTinyMapView {
+    [tester.viewController tinyMapView];
+    [self keyValueObservingExpectationForObject:tester.mapView keyPath:@"contentInset" handler:^BOOL(id observedObject, NSDictionary *change) {
+        XCTAssertEqual(tester.mapView.contentInset.top,
+                       0,
+                       @"map should not have top content inset");
+        XCTAssertEqual(tester.mapView.contentInset.bottom,
+                       0,
+                       @"map should not have bottom content inset");
+        return YES;
+    }];
+    [self waitForExpectationsWithTimeout:5.0 handler:nil];
```

Comment:
I’ll lower this to `1.0`.

This is mainly a buffer against slow CI machines. `waitForExpectationsWithTimeout:handler:` should never add more than a tick or two to the execution time, no matter the timeout length — if exceeds the timeout, it’s a failure and everything stops.

I think xcpretty is swallowing our test output and the elapsed time per test on Bitrise, but locally this test completes in the blink of an eye.